content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+
+### Host enumeration
+Given that I was investigating a single host, and that I had an unfair suspicion of what kind of attack vector I would be looking for, I started with a default script and service version scan.
+
+```
+msf5 > db_nmap -sV -sC -Pn 10.10.201.83
+[*] Nmap: Starting Nmap 7.80 ( https://nmap.org ) at 2020-11-11 01:53 UTC
+[*] Nmap: Nmap scan report for ip-10-10-201-83.eu-west-1.compute.internal (10.10.201.83)
+[*] Nmap: Host is up (0.00053s latency).
+[*] Nmap: Not shown: 991 closed ports
+[*] Nmap: PORT      STATE SERVICE      VERSION
+[*] Nmap: 135/tcp   open  msrpc        Microsoft Windows RPC
+[*] Nmap: 139/tcp   open  netbios-ssn  Microsoft Windows netbios-ssn
+[*] Nmap: 445/tcp   open  microsoft-ds Windows 7 Professional 7601 Service Pack 1 microsoft-ds (workgroup: WORKGROUP)n  tcpwrapped
+[*] Nmap: |_ssl-date: 2020-11-11T01:55:05+00:00; 0s from scanner time.
+[*] Nmap: 49152/tcp open  msrpc        Microsoft Windows RPC
+[*] Nmap: 49153/tcp open  msrpc        Microsoft Windows RPC
+[*] Nmap: 49154/tcp open  msrpc        Microsoft Windows RPC
+[*] Nmap: 49158/tcp open  msrpc        Microsoft Windows RPC
+[*] Nmap: 49160/tcp open  msrpc        Microsoft Windows RPC
+[*] Nmap: MAC Address: 02:B5:23:84:9D:19 (Unknown)
+[*] Nmap: Service Info: Host: JON-PC; OS: Windows; CPE: cpe:/o:microsoft:windows
+[*] Nmap: Host script results:
+[*] Nmap: |_clock-skew: mean: 1h29m59s, deviation: 3h00m00s, median: 0s
+[*] Nmap: |_nbstat: NetBIOS name: JON-PC, NetBIOS user: <unknown>, NetBIOS MAC: 02:b5:23:84:9d:19 (unknown)mb-os-discovery:
+[*] Nmap: |   OS: Windows 7 Professional 7601 Service Pack 1 (Windows 7 Professional 6.1)
+[*] Nmap: |   OS CPE: cpe:/o:microsoft:windows_7::sp1:professional
+[*] Nmap: |   Computer name: Jon-PC
+[*] Nmap: |   NetBIOS computer name: JON-PC\x00
+[*] Nmap: |   Workgroup: WORKGROUP\x00
+[*] Nmap: |_  System time: 2020-11-10T19:54:50-06:00
+[*] Nmap: | smb-security-mode:
+[*] Nmap: |   account_used: guest
+[*] Nmap: |   authentication_level: user
+[*] Nmap: |   challenge_response: supported
+[*] Nmap: |_  message_signing: disabled (dangerous, but default)
+[*] Nmap: | smb2-security-mode:
+[*] Nmap: |   2.02:
+[*] Nmap: |_    Message signing enabled but not required
+[*] Nmap: | smb2-time:
+[*] Nmap: |   date: 2020-11-11T01:54:50
+[*] Nmap: |_  start_date: 2020-11-11T01:51:00
+[*] Nmap: Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .Nmap done: 1 IP address (1 host up) scanned in 140.63 seconds
+```
+
+This initial scan tended to provide further evidence toward my beliefs that this was an eternalblue attack as it looked like the host was a Windows 7 Service Pack 1 machine with SMB available. In addition, this ended up leaking a potential username as the computer's name, which was worth keeping in mind as a potential administrative user going forward. Additionally this pointed out relaxed security setting in the smb service's configuration.
```

Comment:
> This initial scan tended to provide further evidence toward my beliefs that this was an eternalblue attack as it looked like the host was a Windows 7 Service Pack 1 machine with SMB available.

This initial scan began to confirm my suspicions that this was an eternalblue attack since it looked like the host was a Windows 7 Service Pack 1 machine with SMB available.